docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '3'
+networks:
+  db-data-net:
+    driver: bridge
+services:
+  mysql:
+    image: mysql:latest
+    build:
+      context: .
+      dockerfile: Dockerfile.mysql
+    container_name: my-sql
+    ports:
+      - "3306:3306"
+    volumes:
+      - db-data:/var/lib/mysql
+    environment:
+      - MYSQL_ROOT_PASSWORD=1234
+      - MYSQL_DATABASE=app_db
+      - MYSQL_USER=app_user
+      - MYSQL_PASSWORD=1234
+    networks:
+      - db-data-net
+  pythonapp:
+    image: app:1.2
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: todoapp
+    ports:
+      - "8080:8080"
+    environment:
+      - PYTHONUNBUFFERED=1
+    networks:
+      - db-data-net
+    depends_on:
+      - mysql
+    restart: unless-stopped
+volumes:
+  db-data:
```

Comment:
![image](https://github.com/user-attachments/assets/7d5f7f87-f593-4c96-aeb6-2a2910cbfbcf)
Add Eof(End-of-file), need to add a blank line at the end

Reply:
@NazarSheremet I've added blank line